src/OpenSearch/Endpoints/Explain.php

```diff
@@ -24,6 +24,9 @@
 use OpenSearch\Common\Exceptions\RuntimeException;
 use OpenSearch\Endpoints\AbstractEndpoint;
 
+/**
+ * NOTE: This file is autogenerated using util/GenerateEndpoints.php
+ */
```

Comment:
These should probably go on top of the file before `use` and have a WARNING not to update by hand with instructions of how to update via generator.